### DISCUSSION TITLE
Not found in namespace

### DISCUSSION THREAD

**Discussion by jultrem** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3067] [Q&A]
Hi, I have about 110 errors saying that a component cannot be found in the namespace of MaterialDesign. I installed materialDesignThemes in the project.
All the errors are similar to this one.
`The name "TextFieldHintVisibilityConverter" does not exist in the namespace "clr-namespace:MaterialDesignThemes.Wpf.Converters".`

Thanks

**Comment by nicolaihenriksen** [comment]
Did you follow any of the [getting started guides](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit#-getting-started)? My assumption is that you have not loaded the bundled theme or the "defaults".
